src/BUILD

```diff
@@ -562,7 +562,6 @@ filegroup(
         "@android_gmaven_r8_for_testing//jar:file",
         "@android_tools_for_testing//:WORKSPACE",
         "@bazel_skylib//:WORKSPACE",
-        "@com_google_protobuf//:WORKSPACE",
```

Comment:
Why removing this line? This is a hack to make sure we don't download certain dependencies in Bazel's integration tests repeatedly, which reduces Bazel's presubmit time.
See https://cs.opensource.google/bazel/bazel/+/master:src/test/shell/testenv.sh.tmpl;l=270-331?q=%22TEST_REPOSITORY_HOME%22&ss=bazel

Reply:
I had failures (and this was a quick fix). 

I reinvestigated and the cause was renaming BUILD to BUILD.bazel in protobuf. I fixed the WORKSPACE file to patch the right file and now it works.